projectile.el

```diff
@@ -1306,8 +1306,9 @@ VCS is the VCS of the project."
 If `command' is nil or an empty string, return nil.
 This allows commands to be disabled."
   (when (stringp command)
-    (let ((default-directory root))
-      (split-string (shell-command-to-string command) "\0" t))))
+    (let* ((default-directory root)
+           (files (split-string (shell-command-to-string command) "\0" t)))
+      (cl-remove-if-not #'file-exists-p files))))
```

Comment:
The problem with this is that if you get a long list of files the processing is going to be very slow. I've deliberately avoided iterating over the project files in many places for that particular reason.

Reply:
I see. I will propose something else. Thanks for your feedback ;)

Reply:
Just pushed a commit using a different approach: just read what is written to stdout.